src/Kerbero.WebApi/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
```

Comment:
We don't need this controller, right?
Either flag this with a 
```csharp
// TODO remove this
```
or directly remove this file